Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/Dashboard/Booking.hs

```diff
@@ -70,13 +66,6 @@ stuckBookingsCancel merchantShortId req = do
   let stuckPersonIds = stuckRideItems <&> (.driverId)
   let stuckDriverIds = cast @DP.Person @DP.Driver <$> stuckPersonIds
   -- drivers going out of ride, update location from redis to db
-  enableLocationTrackingService <- asks (.enableLocationTrackingService)
-  driverLocations <-
```

Comment:
@chakri642 We have removed this code considering there is no more Redis Going to be there for location updates right so stuck booking does not need to do anything with location updates right ?